test/unit/view/unconnected-droppable-dimension-publisher.spec.js

```diff
@@ -323,359 +321,133 @@ describe('DraggableDimensionPublisher', () => {
-                updateScroll={this.props.updateScroll}
-                shouldPublish={this.props.shouldPublish}
-                updateIsEnabled={this.props.updateIsEnabled}
-              />
-            </ScrollParent>
-          );
-        }
-      }
-
-      type ExecuteArgs = {|
-        droppableRect: ClientRect,
-        scrollParentRect: ClientRect,
-        scrollParentScroll?: Position,
-      |}
-
-      const execute = ({
-        droppableRect,
-        scrollParentRect,
-        scrollParentScroll = origin,
-      }: ExecuteArgs) => {
-        wrapper = mount(
-          <App
-            publish={publish}
-            updateScroll={updateScroll}
-            updateIsEnabled={updateIsEnabled}
-          />
-        );
-
-        const scrollParentNode: HTMLElement = wrapper.getDOMNode();
-
-        if (!scrollParentNode.classList.contains('scroll-parent')) {
-          throw new Error('scroll parent node not obtained correctly');
-        }
-
-        const droppableNode: HTMLElement = scrollParentNode.querySelector('.item');
-
-        scrollParentNode.scrollLeft = scrollParentScroll.x;
-        scrollParentNode.scrollTop = scrollParentScroll.y;
-
-        jest.spyOn(scrollParentNode, 'getBoundingClientRect').mockImplementationOnce(() => scrollParentRect);
-        jest.spyOn(droppableNode, 'getBoundingClientRect').mockImplementationOnce(() => droppableRect);
-
-        jest.spyOn(window, 'getComputedStyle').mockImplementation((el) => {
-          if (el === droppableNode) {
-            return noSpacing;
-          }
-
-          if (el === scrollParentNode) {
-            return {
-              ...noSpacing,
-              overflow: 'auto',
-            };
-          }
-
-          throw new Error('unknown el');
-        });
-
-        wrapper.setProps({
-          shouldPublish: true,
-        });
-      };
-
-      it('should clip the dimension by the size of the scroll parent', () => {
-        const droppableRect: ClientRect = getClientRect({
-          top: 0,
-          bottom: 100,
-          left: 0,
-          right: 100,
-        });
-        // smaller by 10px in every direction
-        const scrollParentRect: ClientRect = getClientRect({
-          top: 10,
-          bottom: 90,
-          left: 10,
-          right: 90,
-        });
-        const expected = getDroppableDimension({
-          id: droppableId,
-          // because it is smaller in every direction
-          // the result will be the scroll parent rect
-          clientRect: scrollParentRect,
-        });
-
-        execute({ droppableRect, scrollParentRect });
-
-        // the trimmed rect
-        expect(publish).toBeCalledWith(expected);
-        expect(publish).toHaveBeenCalledTimes(1);
+      it('should detect scrolling on the droppable', () => {
+        wrapper = mount(<App
+          droppableIsScrollable
+          parentIsScrollable
+          onPublish={(dimension) => {
+            expect(dimension).toBeTruthy();
```

Comment:
I generally do not like doing assertions like this. Any chance this can call a mock and then you check what the mock was called with in the assertion phase?